pkg/core/blockchain.go

```diff
@@ -615,6 +623,17 @@ func (bc *Blockchain) storeBlock(block *block.Block) error {
 	return nil
 }
 
+// SetDebug enables storing info about storage changes
+// before every persist.
+func (bc *Blockchain) SetDebug(debug bool) {
```

Comment:
Why don't we make it a configuration parameter? I think we already know that want these batches when we're creating Blockchain, so it makes little sense adding this switch and yet another non-config variable.

Reply:
I expect `--dump` to work irregardless of what I have set in config. Maybe it should override config settings then?

Reply:
Yeah, that's what I've meant.